### ISSUE TITLE
What do you mean with build the engine from source.

### ISSUE THREAD

**Issue by andrsGutirrz** [wowtomi/godot-mysql-module#5]
Hi there!
Im trying to use your module, but the editors show me an error (look at the pic), I think is for that part you say "build the engine from source", I dont get it what do you mean with that!
Is something Im missing?

EXTRA:
Im working on Ubuntu 18.04 and Godot v3.0.3, downloaded from Godot page
![screenshot from 2018-07-06 12-00-55](https://user-images.githubusercontent.com/16330583/42393649-4599200e-8114-11e8-99a6-87f83ee7beb6.png)

**Comment by christoffersch** [comment]
Did you
1) clone the Godot Engine
2) clone this module to C:\path_to_godot\modules\mysql
3) build the engine from source?

**Comment by andrsGutirrz** [comment]
Yes, I did! I already know how to build from source! and I compiled, but it doesnt work when I call de constructor in a gdscript. Now Im gonna test the connection with an API REST using HTTp object!
